### PR TITLE
ADDED - search id in DB when model instance

### PR DESCRIPTION
**Pull request by YannikFirre**
Hello,

This PR allows you to search for an existing ID in DB rather than using when no example.

Regards

### CODE REVIEW

**Comment by shalvah** [comment]
I don't really understand what's going on here. You have to provide a test case that describes what this is doing. If you don't know how to add a test to the project, you can at least post some example code here.

**Comment by YannikFirre** [comment]
Yes sorry for the lack of information. 

When no example comment is specified, Scribe takes a randomly generated id which can cause a "ModelNotFoundException".

My modification allows to select the 1st id in database if it exists instead of taking a randomly generated id

File: src/Extracting/Strategies/UrlParameters/GetFromLaravelAPI.php [hunk 1]
```diff
@@ -69,11 +69,15 @@ public function __invoke(ExtractedEndpointData $endpointData, array $routeRules)
                     $type = $this->normalizeTypeName($typeName);
                     $parameters[$paramName]['type'] = $type;
 
-                    // If the user explicitly set a `where()` constraint, use that to refine examples
-                    $parameterRegex = $endpointData->route->wheres[$paramName] ?? null;
-                    $example = $parameterRegex
-                        ? $this->castToType($this->getFaker()->regexify($parameterRegex), $type)
-                        : $this->generateDummyValue($type);
+                    $example  = optional($argumentInstance::first())->id;
```
Comment (shalvah) [inline src/Extracting/Strategies/UrlParameters/GetFromLaravelAPI.php:72]: Use `??`, not `optional()`.

Comment (shalvah) [reply src/Extracting/Strategies/UrlParameters/GetFromLaravelAPI.php]: One more thing: please wrap this in a try/catch, so it doesn't crash if the database isn't available.

**Comment by YannikFirre** [comment]
Thanks, Sorry I forgot  =/

**Pull request merged**